app/utils/dictionary/demography.ts

```diff
@@ -4,1139 +4,1423 @@
+    name        : 'Kiribati',
+    code        : 'KI',
+    dialingCode : '+686'
   },
   {
-    name : 'Comoros',
-    code : 'KM'
+    name        : 'Comoros',
+    code        : 'KM',
+    dialingCode : '+269'
   },
   {
-    name : 'Saint Kitts And Nevis',
-    code : 'KN'
+    name        : 'Saint Kitts And Nevis',
+    code        : 'KN',
+    dialingCode : '+1 869'
   },
   {
-    name : 'North Korea',
-    code : 'KP'
+    name        : 'North Korea',
+    code        : 'KP',
+    dialingCode : '+850'
   },
   {
-    name : 'South Korea',
-    code : 'KR'
+    name        : 'South Korea',
+    code        : 'KR',
+    dialingCode : '+82'
   },
   {
-    name : 'Kuwait',
-    code : 'KW'
+    name        : 'Kuwait',
+    code        : 'KW',
+    dialingCode : '+965'
   },
   {
-    name : 'Cayman Islands',
-    code : 'KY'
+    name        : 'Cayman Islands',
+    code        : 'KY',
+    dialingCode : '+345'
   },
   {
-    name : 'Kazakhstan',
-    code : 'KZ'
+    name        : 'Kazakhstan',
+    code        : 'KZ',
+    dialingCode : '+7'
   },
   {
-    name : 'Lao People\'s Democratic Republic',
-    code : 'LA'
+    name        : 'Lao People\'s Democratic Republic',
+    code        : 'LA',
+    dialingCode : '+'
   },
   {
-    name : 'Lebanon',
-    code : 'LB'
+    name        : 'Lebanon',
+    code        : 'LB',
+    dialingCode : '+961'
   },
   {
-    name : 'Saint Lucia',
-    code : 'LC'
+    name        : 'Saint Lucia',
+    code        : 'LC',
+    dialingCode : '+'
   },
   {
-    name : 'Liechtenstein',
-    code : 'LI'
+    name        : 'Liechtenstein',
+    code        : 'LI',
+    dialingCode : '+423'
   },
   {
-    name : 'Sri Lanka',
-    code : 'LK'
+    name        : 'Sri Lanka',
+    code        : 'LK',
+    dialingCode : '+94'
   },
   {
-    name : 'Liberia',
-    code : 'LR'
+    name        : 'Liberia',
+    code        : 'LR',
+    dialingCode : '+231'
   },
   {
-    name : 'Lesotho',
-    code : 'LS'
+    name        : 'Lesotho',
+    code        : 'LS',
+    dialingCode : '+266'
   },
   {
-    name : 'Lithuania',
-    code : 'LT'
+    name        : 'Lithuania',
+    code        : 'LT',
+    dialingCode : '+370'
   },
   {
-    name : 'Luxembourg',
-    code : 'LU'
+    name        : 'Luxembourg',
+    code        : 'LU',
+    dialingCode : '+352'
   },
   {
-    name : 'Latvia',
-    code : 'LV'
+    name        : 'Latvia',
+    code        : 'LV',
+    dialingCode : '+371'
   },
   {
-    name : 'Libya',
-    code : 'LY'
+    name        : 'Libya',
+    code        : 'LY',
+    dialingCode : '+218'
   },
   {
-    name : 'Morocco',
-    code : 'MA'
+    name        : 'Morocco',
+    code        : 'MA',
+    dialingCode : '+212'
   },
   {
-    name : 'Monaco',
-    code : 'MC'
+    name        : 'Monaco',
+    code        : 'MC',
+    dialingCode : '+377'
   },
   {
-    name : 'Moldova',
-    code : 'MD'
+    name        : 'Moldova',
+    code        : 'MD',
+    dialingCode : '+373'       
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5010957)